cfme/common/topology.py

```diff
@@ -58,7 +58,8 @@ class legends(ParametrizedView):  # noqa
 
         @property
         def is_enabled(self):
-            return 'active' in self.browser.get_attribute('class', self.el)
+            el = self.browser.element('./ancestor::kubernetes-topology-icon[1]', self.el)
```

Comment:
**REQUIRED:** This seems specific to kubernetes, what about other topologies?

Reply:
@psav It seems it's common to all providers